src/main/java/com/sobytylnik/hello/MyTest.java

```diff
@@ -0,0 +1,60 @@
+package com.sobytylnik.hello;
+
+
+
+import com.sobytylnik.hello.ExceptionListPackage.EntityAlreadyExistsException;
+import com.sobytylnik.hello.ExceptionListPackage.EntityNotFoundException;
+import org.junit.Test;
+import static org.junit.Assert.*;
+
+import java.util.Optional;
+import java.util.Random;
+import java.util.UUID;
+import java.util.concurrent.ConcurrentHashMap;
+
+public class MyTest {
+    Profile friend = new Profile(new Random().nextLong(), "Sergey", "Apalko", 29);
+    Profile friend1 = new Profile(new Random().nextLong(), "Max", "Luchenko", 29);
+    Profile friend2 = new Profile(new Random().nextLong(), "General", "Panama", 29);
+    Profile checkProfile = new Profile(new Random().nextLong(),"Kolyamba","Kurenkov",27);
+    InMemoryProfileRepository friendList = new InMemoryProfileRepository();
+
+
+    @Test
+    public void testFindById(){
+        friendList.save(friend2);
+
+        assertTrue(friendList.findById(22L).equals(friend2));
+    }
+
+    @Test
+    public void testSave(){
+        friendList.save(friend);
+        friendList.save(friend1);
+        assertEquals(friendList.findById(13L), friend1);
+    }
+
+    @Test(expected = EntityAlreadyExistsException.class)
+    public void testSave2(){
```

Comment:
Try to add more descriptive test names. For example, `whenSaveExistentUser_expectException`.
https://dzone.com/articles/7-popular-unit-test-naming